### PR TITLE
Add doc files for syncable exports

### PR DESCRIPTION
**Pull request by bangelic**
Three new document files have been added describing the procedure for
syncable exports. This is mirroring PR #1691. This is the same content
generated from that PR and with some grammatical and spelling fixes.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.4/Katello 4.6
* [x] Foreman 3.3/Katello 4.5
* [ ] Foreman 3.2/Katello 4.4
* [ ] Foreman 3.1/Katello 4.3
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.3 or older.

### CODE REVIEW

File: guides/common/modules/proc_exporting-a-content-view-version-in-syncable-format.adoc [hunk 1]
```diff
@@ -0,0 +1,82 @@
+[id="Exporting_a_Content_View_Version_in_a_Syncable_Format_{context}"]
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-content-view-version-in-syncable-format.adoc:1]: please ensure the file name matches the title.

File: guides/common/modules/proc_exporting-a-content-view-version-in-syncable-format.adoc [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+# hammer content-view version list \
+--content-view="_My_Content_View_" \
+--organization="_My_Organization_"
+---|----------|---------|-------------|-----------------------
+ID | NAME     | VERSION | DESCRIPTION | LIFECYCLE ENVIRONMENTS
+---|----------|---------|-------------|-----------------------
+5  | view 3.0 | 3.0     |             | Library
+4  | view 2.0 | 2.0     |             |
+3  | view 1.0 | 1.0     |             |
+---|----------|---------|-------------|----------------------
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-content-view-version-in-syncable-format.adoc:62]:
```suggestion
```

File: guides/common/modules/proc_exporting-a-content-view-version-in-syncable-format.adoc [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+. Get the version number of desired version.
+The following example targets version `1.0` for export:
++
+[options="nowrap" subs="+quotes"]
+----
+# hammer content-export complete version \
+--content-view="_Content_View_Name_" \
+--version=1.0 \
+--organization="_My_Organization_"
+ --format=syncable
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-content-view-version-in-syncable-format.adoc:75]:
```suggestion
--format=syncable
```

File: guides/common/modules/proc_exporting-a-content-view-version-in-syncable-format.adoc [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+.Export a Content View version
+. Get the version number of desired version.
+The following example targets version `1.0` for export:
++
+[options="nowrap" subs="+quotes"]
+----
+# hammer content-export complete version \
+--content-view="_Content_View_Name_" \
+--version=1.0 \
+--organization="_My_Organization_"
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-content-view-version-in-syncable-format.adoc:74]:
```suggestion
--organization="_My_Organization_" \
```

File: guides/common/modules/proc_exporting-a-content-view-version-in-syncable-format.adoc [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+ --format=syncable
+----
+. Optional: Verify that the exported content is located in the export directory:
++
+[options="nowrap" subs="+quotes"]
+----
+# ls -lh /var/lib/pulp/exports/_My_Organization_/_Content_View_Name_/1.0/2021-02-25T18-59-26-00-00/
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-content-view-version-in-syncable-format.adoc:81]:
```suggestion
# ls -lh /var/lib/pulp/exports/_My_Organization_/_My_Content_View_Name_/1.0/2021-02-25T18-59-26-00-00/
```

File: guides/common/modules/proc_exporting-a-repository-in-syncable-format.adoc [hunk 6]
```diff
@@ -0,0 +1,65 @@
+[id="Exporting_a_Repository_in_a_Syncable_Format_{context}"]
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-repository-in-syncable-format.adoc:1]: same as above.

File: guides/common/modules/proc_exporting-a-repository-in-syncable-format.adoc [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+* RPM repositories
+* Kickstart repositories
+* File repositories
+
+You cannot export Ansible and Docker content.
+
+ifndef::satellite[]
+You cannot export DEB repositories
+endif::[]
+from {ProjectServer}.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-repository-in-syncable-format.adoc:34]:
```suggestion
```

File: guides/common/modules/proc_exporting-a-repository-in-syncable-format.adoc [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+You can export the following content in a syncable format from {ProjectServer}:
+
+* RPM repositories
+* Kickstart repositories
+* File repositories
+
+You cannot export Ansible and Docker content.
+
+ifndef::satellite[]
+You cannot export DEB repositories
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-repository-in-syncable-format.adoc:32]:
```suggestion
You cannot export DEB repositories from {ProjectServer}.
```

File: guides/common/modules/proc_exporting-a-repository-in-syncable-format.adoc [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+endif::[]
+from {ProjectServer}.
+
+The export contains the following data:
+
+* Directories containing the RPMs, *listing* files and metadata of the repository in yum format that can be used to synchronize in the importing {ProjectServer}.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-repository-in-syncable-format.adoc:38]: I am unsure if we really need a list here; maybe reword? Or split into several list items?

I think a single sentence is best because the exported parts depend on the content type; so maybe your single sentence should be prefixed with "For Yum content, ..." or similar.

File: guides/common/modules/proc_exporting-a-repository-in-syncable-format.adoc [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+. Optional: Verify that the exported content is located in the export directory:
++
+[options="nowrap" subs="+quotes"]
+----
+# ls -lh /var/lib/pulp/exports/_My_Organization_/Export-_My_Repository_/1.0/2021-03-02T03-35-24-00-00
+total 0
+drwxr-xr-x. 3 pulp pulp 33 Aug 26 09:16 content
+
+# du -sh /var/lib/pulp/exports/_My_Organization_/Export-_My_Repository_/1.0/2021-03-02T03-35-24-00-00
+439M
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-repository-in-syncable-format.adoc:64]:
```suggestion
# du -sh /var/lib/pulp/exports/_My_Organization_/Export-_My_Repository_/1.0/2021-03-
```

File: guides/common/modules/proc_exporting-a-repository-in-syncable-format.adoc [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+ --organization="_My_Organization_" \
+ --product="_My_Product_" \
+ --name="_My_Repository_"\
+ --format=syncable
+----
+. Optional: Verify that the exported content is located in the export directory:
++
+[options="nowrap" subs="+quotes"]
+----
+# ls -lh /var/lib/pulp/exports/_My_Organization_/Export-_My_Repository_/1.0/2021-03-02T03-35-24-00-00
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-repository-in-syncable-format.adoc:59]:
```suggestion
```

File: guides/common/modules/proc_exporting-a-repository.adoc [hunk 12]
```diff
@@ -9,11 +9,11 @@ You can export the following content from {ProjectServer}:
...
 * Kickstart repositories
 * Yum repositories
-* file repositories
+* File repositories
+* Docker content
 
-You cannot export Docker content
 ifndef::satellite[]
-nor DEB repositories
+You cannot export DEB repositories
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-repository.adoc:16]:
```suggestion
You cannot export DEB repositories from {ProjectServer}.
```

File: guides/common/modules/proc_exporting-a-repository.adoc [hunk 13]
```diff
@@ -9,11 +9,11 @@ You can export the following content from {ProjectServer}:
...
 endif::[]
 from {ProjectServer}.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-repository.adoc:18]:
```suggestion
```

File: guides/common/modules/proc_exporting-the-library-environment-in-syncable-format.adoc [hunk 14]
```diff
@@ -0,0 +1,63 @@
+[id="Exporting_the_Library_Environment_in_a_Syncable_Format_{context}"]
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-the-library-environment-in-syncable-format.adoc:1]: same as above.

File: guides/common/modules/proc_exporting-the-library-environment-in-syncable-format.adoc [hunk 15]
```diff
@@ -0,0 +1,63 @@
...
+
+You cannot export Ansible and Docker content.
+
+ifndef::satellite[]
+You cannot export DEB repositories.
+endif::[]
+
+The export contains the following data:
+
+* Directories containing the RPMs, *listing* files and metadata of each repository in yum format that can be used to synchronize in the importing {ProjectServer}.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-the-library-environment-in-syncable-format.adoc:38]: same as above.

File: guides/common/modules/proc_exporting-the-library-environment-in-syncable-format.adoc [hunk 16]
```diff
@@ -0,0 +1,63 @@
...
+----
+# hammer content-export complete library \
+--organization="_My_Organization_" \
+--format=syncable
+----
+. Optional: Verify that the exported content is located in the export directory:
++
+[options="nowrap" subs="+quotes"]
+----
+# ls -lh /var/lib/pulp/exports/_My_Organization_/Export-Library/1.0/2021-02-25T18-59-26-00-00/
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-the-library-environment-in-syncable-format.adoc:62]: Maybe also use `du` as above?

**Review by maximiliankolb — CHANGES_REQUESTED** [review]
During review, I felt like I have reviewed a very similar PR like this before. Fixing a PR is always easier for others than creating a new one.

File: guides/common/modules/proc_exporting-a-content-view-version-in-syncable-format.adoc [hunk 17]
```diff
@@ -0,0 +1,82 @@
...
+
+ifdef::satellite[]
+You can then serve the generated content via a local webserver on the importing {ProjectServer} or in another {ProjectServer} organization.
+
+You cannot directly import Syncable Format exports. Instead, on the importing {ProjectServer} you must:
+
+* Copy the generated content to an HTTP/HTTPS webserver that is accessible to importing {ProjectServer}.
+* Update your CDN configuration to *Custom CDN*.
+* Set the CDN URL to point to the web server.
+* Optionally set a SSL CA Credential if the web server requires it.
```
Comment (mjivraja) [inline guides/common/modules/proc_exporting-a-content-view-version-in-syncable-format.adoc:14]:
```suggestion
* Optionally set an SSL/TLS CA Credential if the web server requires it.
```
Learned this from Vale. It should be technically correct.

File: guides/common/modules/proc_exporting-a-content-view-version-in-syncable-format.adoc [hunk 18]
```diff
@@ -0,0 +1,82 @@
+[id="Exporting_a_Content_View_Version_in_a_Syncable_Format_{context}"]
+= Exporting a Content View Version in a Syncable Format
+
+You can export a version of a Content View to a syncable format that you can use to create your custom CDN and synchronize the content from the custom CDN over HTTP/HTTPS.
+
+ifdef::satellite[]
+You can then serve the generated content via a local webserver on the importing {ProjectServer} or in another {ProjectServer} organization.
```
Comment (mjivraja) [inline guides/common/modules/proc_exporting-a-content-view-version-in-syncable-format.adoc:7]:
```suggestion
You can then serve the generated content through a local webserver on the importing {ProjectServer} or in another {ProjectServer} organization.
```

File: guides/common/modules/proc_exporting-a-content-view-version-in-a-syncable-format.adoc [hunk 19]
```diff
@@ -0,0 +1,75 @@
+[id="Exporting_a_Content_View_Version_in_a_Syncable_Format_{context}"]
+= Exporting a Content View Version in a Syncable Format
+
+You can export a version of a Content View to a syncable format that you can use to create your custom CDN and synchronize the content from the custom CDN over HTTP/HTTPS.
+
+ifdef::satellite[]
+You can then serve the generated content through a local webserver on the importing {ProjectServer} or in another {ProjectServer} organization.
+
+You cannot directly import Syncable Format exports. Instead, on the importing {ProjectServer} you must:
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-content-view-version-in-a-syncable-format.adoc:9]:
```suggestion
You cannot directly import Syncable Format exports.
Instead, on the importing {ProjectServer} you must:
```

File: guides/common/modules/proc_exporting-a-content-view-version-in-a-syncable-format.adoc [hunk 20]
```diff
@@ -0,0 +1,75 @@
...
+
+* Copy the generated content to an HTTP/HTTPS webserver that is accessible to importing {ProjectServer}.
+* Update your CDN configuration to *Custom CDN*.
+* Set the CDN URL to point to the web server.
+* Optionally set an SSL/TLS CA Credential if the web server requires it.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-content-view-version-in-a-syncable-format.adoc:14]:
```suggestion
* Optional: Set an SSL/TLS CA Credential if the web server requires it.
```

File: guides/common/modules/proc_exporting-a-content-view-version-in-a-syncable-format.adoc [hunk 21]
```diff
@@ -0,0 +1,75 @@
...
+
+You cannot export Ansible and Docker content.
+
+ifndef::satellite[]
+You cannot export DEB repositories.
+endif::[]
+
+The export contains the following data:
+
+* Directories containing the RPMs, *listing* files and metadata of each repository in yum format that can be used to synchronize in the importing {ProjectServer}.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-content-view-version-in-a-syncable-format.adoc:40]:
```suggestion
* Directories containing the RPMs, *listing* files and metadata of each repository in yum format that can be used to synchronize in the importing {ProjectServer}.
```

Would it make sense to reword "RPMs" to "packages"? Or even something more generic that also applies to file repositories?

File: guides/common/modules/proc_exporting-a-content-view-version-in-a-syncable-format.adoc [hunk 22]
```diff
@@ -0,0 +1,75 @@
...
+
+.Prerequisites
+* Ensure that you set download policy to *Immediate* for all repositories within the Content View you export.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-content-view-version-in-a-syncable-format.adoc:43]:
```suggestion
* Ensure that you set the download policy to *Immediate* for all repositories within the Content View you export.
```

File: guides/common/modules/proc_exporting-a-content-view-version-in-a-syncable-format.adoc [hunk 23]
```diff
@@ -0,0 +1,75 @@
...
+For more information, see xref:Download_Policies_Overview_{context}[].
+* Ensure that you synchronize products you export to the required date.
+* Ensure that the user exporting the content has the `Content Exporter` role.
+
+.To Export a Content View Version
+. List versions of the Content View that are available for export:
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-content-view-version-in-a-syncable-format.adoc:49]:
```suggestion
* List versions of the Content View that are available for export:
```

IMHO: if it's only one list element, we go with an unordered list.

File: guides/common/modules/proc_exporting-a-repository-in-a-syncable-format.adoc [hunk 24]
```diff
@@ -0,0 +1,59 @@
+[id="Exporting_a_Repository_in_a_Syncable_Format_{context}"]
+= Exporting a Repository in a Syncable Format
+
+You can export the content of a repository in the Library environment of an organization of an organization to a syncabe format that you can use to create your custom CDN and synchronize the content from the custom CDN over HTTP/HTTPS.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-repository-in-a-syncable-format.adoc:4]:
```suggestion
You can export the content of a repository in the Library environment of an organization to a syncabe format that you can use to create your custom CDN and synchronize the content from the custom CDN over HTTP/HTTPS.
```

File: guides/common/modules/proc_exporting-a-repository-in-a-syncable-format.adoc [hunk 25]
```diff
@@ -0,0 +1,59 @@
...
+
+ifdef::satellite[]
+You can then serve the generated content using a local webserver on the importing {ProjectServer} or in another {ProjectServer} organization.
+
+You cannot directly import Syncable Format exports. Instead on the importing {ProjectServer} you would need to:
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-repository-in-a-syncable-format.adoc:9]:
```suggestion
You cannot directly import Syncable Format exports.
Instead on the importing {ProjectServer} you must:
```

I think you said "must" in the other file.

File: guides/common/modules/proc_exporting-a-repository-in-a-syncable-format.adoc [hunk 26]
```diff
@@ -0,0 +1,59 @@
...
+
+* Copy the generated content to an HTTP/HTTPS webserver that is accessible to importing {ProjectServer}.
+* Update your CDN configuration to *Custom CDN*.
+* Set the CDN URL to point to the web server.
+* Optionally set a SSL CA Credential if the web server requires it.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-repository-in-a-syncable-format.adoc:14]:
```suggestion
* Optional: Set a SSL CA Credential if the web server requires it.
```

File: guides/common/modules/proc_exporting-a-repository-in-a-syncable-format.adoc [hunk 27]
```diff
@@ -0,0 +1,59 @@
...
+* Enable the Repository.
+* Synchronize the Repository.
+endif::[]
+
+ifndef::satellite[]
+You can use the generated content to create the same repository in another {ProjectServer} or in another {ProjectServer} organization via content import.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-repository-in-a-syncable-format.adoc:20]:
```suggestion
You can use the generated content to create the same repository in another {ProjectServer} or in another {ProjectServer} organization using content import.
```

According to IBM style guide, we're meant to avoid such latin words.

File: guides/common/modules/proc_exporting-a-repository-in-a-syncable-format.adoc [hunk 28]
```diff
@@ -0,0 +1,59 @@
...
+ifndef::satellite[]
+You cannot export DEB repositories from {ProjectServer}.
+endif::[]
+
+The export contains the following data:
+
+Directories containing the RPMs, *listing* files and metadata of the repository in yum format that can be used to synchronize in the importing {ProjectServer}.
+
+.Prerequisite
+* Ensure that you set download policy to *Immediate* for the repository within the Library lifecycle environment you export.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-repository-in-a-syncable-format.adoc:40]:
```suggestion
* Ensure that you set the download policy to *Immediate* for the repository within the Library lifecycle environment you export.
```

File: guides/common/modules/proc_exporting-a-repository-in-a-syncable-format.adoc [hunk 29]
```diff
@@ -0,0 +1,59 @@
...
+
+.Procedure
+. Export a repository using the repository name or ID:
++
+[options="nowrap" subs="+quotes"]
+----
+# hammer content-export complete repository \
+ --organization="_My_Organization_" \
+ --product="_My_Product_" \
+ --name="_My_Repository_"\
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-repository-in-a-syncable-format.adoc:51]:
```suggestion
 --name="_My_Repository_" \
```

File: guides/common/modules/proc_exporting-a-repository-in-a-syncable-format.adoc [hunk 30]
```diff
@@ -0,0 +1,59 @@
...
+* Ensure that you set download policy to *Immediate* for the repository within the Library lifecycle environment you export.
+For more information, see xref:Download_Policies_Overview_{context}[].
+
+.Procedure
+. Export a repository using the repository name or ID:
++
+[options="nowrap" subs="+quotes"]
+----
+# hammer content-export complete repository \
+ --organization="_My_Organization_" \
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-repository-in-a-syncable-format.adoc:49]:
```suggestion
--organization="_My_Organization_" \
```

We mostly avoid whitespace before multi line commands. Please adjust all affected lines.

File: guides/common/modules/proc_exporting-the-library-environment-in-a-syncable-format.adoc [hunk 31]
```diff
@@ -0,0 +1,63 @@
+[id="Exporting_the_Library_Environment_in_a_Syncable_Format_{context}"]
+= Exporting the Library Environment in a Syncable Format
+
+You can export contents of all Yum repositories in the Library environment of an organization to a syncable format that you can use to create your custom CDN and synchronize the content from the custom CDN over HTTP/HTTPS.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-the-library-environment-in-a-syncable-format.adoc:4]: Is `Yum` correct here? the list in line 26-28 also contains ks and file repos.

File: guides/common/modules/proc_exporting-the-library-environment-in-a-syncable-format.adoc [hunk 32]
```diff
@@ -0,0 +1,63 @@
...
+
+ifdef::satellite[]
+You can then serve the generated content via a local webserver on the importing  {ProjectServer} or in another {ProjectServer} organization.
+
+You cannot directly import Syncable Format exports. Instead on the importing {ProjectServer} you would need to
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-the-library-environment-in-a-syncable-format.adoc:9]:
```suggestion
You cannot directly import Syncable Format exports.
Instead on the importing {ProjectServer} you would need to
```

I am wondering if using a snippet here makes sense.


Above, I think you used "must" followed by a colon.

File: guides/common/modules/proc_exporting-the-library-environment-in-a-syncable-format.adoc [hunk 33]
```diff
@@ -0,0 +1,63 @@
...
+* Copy the generated content to an HTTP/HTTPS webserver that is accessible to importing {ProjectServer}.
+* Update your CDN configuration to *Custom CDN*.
+* Set the CDN URL to point to the web server.
+* Optionally set a SSL CA Credential if the web server requires it.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-the-library-environment-in-a-syncable-format.adoc:13]:
```suggestion
* Optional: Set a SSL CA Credential if the web server requires it.
```

File: guides/common/modules/proc_exporting-the-library-environment-in-a-syncable-format.adoc [hunk 34]
```diff
@@ -0,0 +1,63 @@
...
+* Enable the Repository.
+* Synchronize the Repository.
+endif::[]
+
+ifndef::satellite[]
+You can use the generated content to create the same repositories in another {ProjectServer} or in another {ProjectServer} organization via content import.
+On import, the products and repositories are mirrored on your downstream {ProjectServer}.
+endif::[]
+
+
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-the-library-environment-in-a-syncable-format.adoc:23]:
```suggestion
```

File: guides/common/modules/proc_exporting-the-library-environment-in-a-syncable-format.adoc [hunk 35]
```diff
@@ -0,0 +1,63 @@
...
+endif::[]
+
+The export contains the following data:
+
+Directories containing the RPMs, *listing* files and metadata of each repository in yum format that can be used to synchronize in the importing {ProjectServer}.
+
+.Prerequisites
+
+To export the contents of the Library lifecycle environment of the organization, ensure that where you want to export {ProjectServer} meets the following conditions:
+
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-the-library-environment-in-a-syncable-format.adoc:43]:
```suggestion
```

I'd leave it out; having the header "Prerequisites" is IMHO good enough.

File: guides/common/modules/proc_exporting-the-library-environment-in-a-syncable-format.adoc [hunk 36]
```diff
@@ -0,0 +1,63 @@
...
+* Ensure that you set download policy to *Immediate* for all repositories within the Library lifecycle environment you export.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-the-library-environment-in-a-syncable-format.adoc:44]:
```suggestion
* Ensure that you set the download policy to *Immediate* for all repositories within the Library lifecycle environment you export.
```

**Review by maximiliankolb — COMMENTED** [review]
Made a couple of comments; the rest and the esp. structure look good to me!

File: guides/common/modules/proc_exporting-the-library-environment-in-a-syncable-format.adoc (hunk 31 above)
Comment (bangelic) [reply guides/common/modules/proc_exporting-the-library-environment-in-a-syncable-format.adoc:4]: @parthaa Can you confirm?

**Comment by maximiliankolb** [comment]
Hey @bangelic I've seen you've resolved all but one of my suggestions but did not push any changes.

**Comment by bangelic** [comment]
> Hey @bangelic I've seen you've resolved all but one of my suggestions but did not push any changes.

Still waiting to hear from @parthaa about using `Yum`.

File: guides/common/modules/proc_exporting-the-library-environment-in-a-syncable-format.adoc (hunk 31 above)
Comment (parthaa) [reply guides/common/modules/proc_exporting-the-library-environment-in-a-syncable-format.adoc:4]: Yes Yum, KS and file repos

**Comment by bangelic** [comment]
@maximiliankolb Updated and committed after feedback from Partha.

File: guides/common/modules/proc_exporting-a-content-view-version-in-a-syncable-format.adoc [hunk 37]
```diff
@@ -0,0 +1,76 @@
...
+* List versions of the Content View that are available for export:
++
+[subs="+quotes"]
+----
+# hammer content-view version list \
+--content-view="_My_Content_View_" \
+--organization="_My_Organization_"
+----
+
+.Export a Content View version
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-content-view-version-in-a-syncable-format.adoc:59]: I am unsure about this heading and the heading in line 49. Maybe use a single heading `.Procedure` instead?

File: guides/common/modules/proc_exporting-a-repository-in-a-syncable-format.adoc [hunk 38]
```diff
@@ -0,0 +1,60 @@
...
+ifdef::satellite[]
+You can then serve the generated content using a local webserver on the importing {ProjectServer} or in another {ProjectServer} organization.
+
+You cannot directly import Syncable Format exports.
+Instead on the importing {ProjectServer} you must:
+
+* Copy the generated content to an HTTP/HTTPS webserver that is accessible to importing {ProjectServer}.
+* Update your CDN configuration to *Custom CDN*.
+* Set the CDN URL to point to the web server.
+* Optional: Set an SSL CA Credential if the web server requires it.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-repository-in-a-syncable-format.adoc:15]: In the other document, you write "SSL/TLS CA Credential". Maybe the duplicate block for Satellite should be moved to a snippet to avoid duplication.

The snippet may contain line 6 through 38.

File: guides/common/modules/proc_exporting-the-library-environment-in-a-syncable-format.adoc [hunk 39]
```diff
@@ -0,0 +1,62 @@
...
+ifndef::satellite[]
+You cannot export DEB repositories from {ProjectServer}.
+endif::[]
+
+The export contains the following data:
+
+Directories containing the RPMs, *listing* files and metadata of each repository in yum format that can be used to synchronize in the importing {ProjectServer}.
+
+.Prerequisites
+
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-the-library-environment-in-a-syncable-format.adoc:42]:
```suggestion
```

**Review by maximiliankolb — COMMENTED** [review]
Some small suggestions, rest LGTM!

**Comment by bangelic** [comment]
@maximiliankolb Took care of those suggestions and created a snippet.

File: guides/common/modules/proc_exporting-a-repository-in-a-syncable-format.adoc [hunk 40]
```diff
@@ -0,0 +1,28 @@
+[id="Exporting_a_Repository_in_a_Syncable_Format_{context}"]
+= Exporting a Repository in a Syncable Format
+
+You can export the content of a repository in the Library environment of an organization to a syncable format that you can use to create your custom CDN and synchronize the content from the custom CDN over HTTP/HTTPS.
+
+include::snip_generating-content.adoc[]
+
+.Prerequisites
```
Comment (maximiliankolb) [inline guides/common/modules/proc_exporting-a-repository-in-a-syncable-format.adoc:8]:
```suggestion
.Prerequisite
```

File: guides/common/modules/snip_generating-content.adoc [hunk 41]
```diff
@@ -0,0 +1,35 @@
...
+
+ifndef::satellite[]
+You can use the generated content to create the same repository in another {ProjectServer} or in another {ProjectServer} organization using content import.
+On importing the exported archive, a regular Content View is created or updated on your downstream {ProjectServer}.
+For more information, see xref:Importing_a_Content_View_Version_{context}[].
+endif::[]
+
+You can export the following content in a syncable format from {ProjectServer}:
+
+* RPM repositories
```
Comment (maximiliankolb) [inline guides/common/modules/snip_generating-content.adoc:23]:
```suggestion
* Yum repositories
```

File: guides/common/modules/snip_generating-content.adoc [hunk 42]
```diff
@@ -0,0 +1,35 @@
...
+* Kickstart repositories
+* File repositories
+
+You cannot export Ansible and Docker content.
+
+ifndef::satellite[]
+You cannot export DEB repositories from {ProjectServer}.
+endif::[]
```
Comment (maximiliankolb) [inline guides/common/modules/snip_generating-content.adoc:31]:
```suggestion
You cannot export DEB repositories from {ProjectServer}.
```

File: guides/common/modules/snip_generating-content.adoc [hunk 43]
```diff
@@ -0,0 +1,35 @@
...
+
+The export contains the following data:
+
+* Directories containing the packages, *listing* files and metadata of the repository in yum format that can be used to synchronize in the importing {ProjectServer}.
```
Comment (maximiliankolb) [inline guides/common/modules/snip_generating-content.adoc:35]:
```suggestion
The export contains directories with the packages, *listing* files, and metadata of the repository in Yum format that can be used to synchronize in the importing {ProjectServer}.
```

**Review by maximiliankolb — CHANGES_REQUESTED** [review]
Some small comments, rest LGTM!

File: guides/common/modules/snip_generating-content.adoc [hunk 44]
```diff
@@ -0,0 +1,31 @@
...
+
+You can export the following content in a syncable format from {ProjectServer}:
+
+* Yum repositories
+* Kickstart repositories
+* File repositories
+
+You cannot export Ansible and Docker content.
+
+You cannot export DEB repositories from {ProjectServer}.
```
Comment (maximiliankolb) [inline guides/common/modules/snip_generating-content.adoc:29]:
```suggestion
You cannot export Ansible, DEB, and Docker content.
```

**Review by maximiliankolb — COMMENTED** [review]
One small suggestion, rest LGTM!

**Comment by bangelic** [comment]
@maximiliankolb Done!

**Review by maximiliankolb — APPROVED** [review]
Thanks Brian, LGTM!

**Comment by maximiliankolb** [comment]
Merged to master and cherry-picked: :heavy_check_mark: 
   c665354a9..9b0d94c2f  3.5 -> 3.5
   d7f27b566..252dbed7c  3.4 -> 3.4
   052d480a0..7663ea301  3.3 -> 3.3
